tests/unit/src/main/scala/scala/meta/internal/metals/debug/Debugger.scala

```diff
@@ -122,20 +122,20 @@ final class Debugger(server: RemoteServer)(implicit ec: ExecutionContext) {
 
   def stackTrace(thread: Long): Future[StackTraceResponse] = {
```

Comment:
I would change `thread: Long` -> `threadId -> Int` (threadId is Int now)

Reply:
Do you think it also makes sense to then change both `def scopes` and also `def variables` to be `Int`s as well?

Reply:
yes because right now it is: Int from LSP -> convert to Long -> modify other thing -> convert to Int -> put into LSP (we could simply avoid `convert` step)

Reply:
Sounds good. I changed these as well.